tests/test_admin.py

```diff
@@ -11,7 +11,7 @@
 patch_admin_extra_urls_decorators()
 
 from unicef_security import admin
-from unicef_security.models import User, Region
+from unicef_security.models import User, Region, BusinessArea
```

Comment:
please use isort

Reply:
Ok, i will fix these.
Is there a reason why isort is not called on the tests directory in the tox.ini file? (same question for flake8)